docs/configuration.md

```diff
@@ -87,6 +88,8 @@ module.exports = {
 
 `browserLanguage` - languge of browser `default: en-GB` ISO 639-1 codes
 
+`chromeOptions` - empty array, where user can pass desired chromeOptions arguments,g list of [available switches](https://peter.sh/experiments/chromium-command-line-switches/)
```

Comment:
As in the comment below, I think, the better name for this array might be `chromeOptionsAdditionalArgs` or something familiar. It's import for a user to understand that this is an array that extends some existing chromeOptions.args

-----
Typos:
- `empty array` > `an empty array` ;) 
- `arguments,g` > `arguments, g`